spectrum-offchain/src/transaction.rs

```diff
@@ -9,15 +9,15 @@ pub struct TransactionCandidate {
     /// inputs, that are not going to be spent by transaction, but will be reachable from inputs
     /// scripts. `dataInputs` scripts will not be executed, thus their scripts costs are not
     /// included in transaction cost and they do not contain spending proofs.
-    pub data_inputs: Option<TxIoVec<DataInput>>,
+    pub data_inputs: Option<TxIoVec<ErgoBox>>,
```

Comment:
Replaced `DataInput` with its associated box, because the box is needed in `TransactionContext` for signing. Note that we can always recover the `DataInput` from the box.